shared/src/metabase/mbql/schema.cljc

```diff
@@ -474,6 +484,9 @@
    (partial is-clause? arithmetic-expressions)
    (s/recursive #'ArithmeticExpression)
 
+   (partial is-clause? date-extract-functions)
+   (s/recursive #'DatetimeExpression)
```

Comment:
this will allow us to do `[:- [:get-year field] 3` or `[:abs [:get-year field]]`